### ISSUE TITLE
Rename ArchLinux AUR packages to -bin

### ISSUE THREAD

**Issue by benr** [mondoohq/installer#378]
A request on the [cnquery AUR page](https://aur.archlinux.org/packages/cnquery): 

`Please change this package to build from source or reupload as -bin.`

**Comment by benr** [comment]
@atomic111 @arlimus @jaym As ArchLinux users, your opinion on this?

**Comment by jaym** [comment]
that is the appropriate pattern. generally, not `-bin` means its going to be compiled from source

**Comment by atomic111** [comment]
yes, bin should be ok, because we deliver the cnquery/ cnspec binary with the aur package and do not compile it doing the package installation
